scalafix-tests/output/src/main/scala-2/test/explicitResultTypes/ImplicitMembersWithSkipSimpleDefinition.scala

```diff
@@ -0,0 +1,13 @@
+package test.explicitResultTypes
+
+object ImplicitMembersWithSkipSimpleDefinition {
+
+    val x = 42
+
+    implicit var y = 43
```

Comment:
Literal definitions are skipped, regardless if it's implicit or not, as expected